source/scss/components/_quicksearch.scss

```diff
@@ -0,0 +1,52 @@
+.qs-with-background {
+  bottom: 150px;
+  margin-bottom: -100px;
+  position: relative;
+  @media(max-width: 1279px) {
```

Comment:
you should use mixins for all the breakpoints

Reply:
was fixed